source.yml

```diff
@@ -3,4 +3,18 @@ name: ric
 imports:
     - github: rock-core/package_set
 
+######################
```

Comment:
Here a `version_control:` is missing (otherwise, autoproj thinks the following entries are part of the `imports:` list

Reply:
Thanks! added it